### PR TITLE
functions and array challenges

### PR DESCRIPTION
**Pull request by adrianagithub**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -45,9 +48,15 @@ console.log('hello world')
  *
  * Example Output: "$49.90"
  */
+ var price = 5.98;
+ var quantity = 4;
+ var total = price * quantity;
+ console.log(Math.round(total));
```
Comment (sinancengiz) [inline index.js:54]: in line 54 Math.round() make a number rounded. we can use .toFixed(2) to get 2 decimal https://www.w3schools.com/JSREF/jsref_tofixed.asp

File: index.js [hunk 2]
```diff
@@ -61,8 +70,19 @@ console.log('hello world')
  *
  * Example Output: "Aww! It's not sunny outside"
  */
-
 // TODO: ADD CODE HERE
+var weather ="Sunny";
+weather.toLowerCase;
+var message =String;
+if (weather = "sunny")
```
Comment (sinancengiz) [inline index.js:77]: in line 77 to check equality , we can use "==" or  "==="

File: index.js [hunk 3]
```diff
@@ -85,11 +108,17 @@ console.log('hello world')
  * 2. Add a return statement in your function which returns a random string
  * 3. Use `console.log` to print out the output of `myArrowFunction()`
  *
- * Example Output: "I called an arrow function!"
- */
-
 // TODO: ADD CODE HERE
-
+*/
```
Comment (sinancengiz) [inline index.js:112]: Question 6 require using a arrow function Example arrow function 

hello = () => {
  return "Hello World!";
}

**Review by sinancengiz — APPROVED** [review]
Hi Adriana, 

I  added a few comments, good job

**Comment by adrianagithub** [comment]
Thank u Sinan 🙏🏽

**Comment by sinancengiz** [comment]
> Thank u Sinan 🙏🏽
> […](#)
> Sent from my iPhone
> On Jul 17, 2020, at 10:12 PM, sinan cengiz ***@***.***> wrote: ﻿ @sinancengiz approved this pull request. Hi Adriana, I added a few comments, good job In index.js: > @@ -45,9 +48,15 @@ console.log('hello world') * * Example Output: "$49.90" */ + var price = 5.98; + var quantity = 4; + var total = price * quantity; + console.log(Math.round(total)); in line 54 Math.round() make a number rounded. we can use .toFixed(2) to get 2 decimal https://www.w3schools.com/JSREF/jsref_tofixed.asp In index.js: > // TODO: ADD CODE HERE +var weather ="Sunny"; +weather.toLowerCase; +var message =String; +if (weather = "sunny") in line 77 to check equality , we can use "==" or "===" In index.js: > // TODO: ADD CODE HERE - +*/ Question 6 require using a arrow function Example arrow function hello = () => { return "Hello World!"; } — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe.

**Pull request closed without merging**